lib/src/test_flow_control_helper.dart

```diff
@@ -81,7 +81,7 @@ class TestFlowControlHelper {
           widgetless: true,
           type: null,
         ),
-        testRunnerStepBuilder: ConditionalStep.fromDynamic,
+        testRunnerStepBuilder: AssertVariableValueStep.fromDynamic,
```

Comment:
Oh crap, good catch.  I'll make this fix today so it doesn't have to be associated with this PR.